src/test/java/co/alphavantage/CurrencyExchangeTest.java

```diff
@@ -0,0 +1,29 @@
+package co.alphavantage;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+import org.hamcrest.core.IsEqual;
+import static org.hamcrest.CoreMatchers.equalTo;
+import org.junit.Test;
+
+import co.alphavantage.output.exchange.CurrencyExchange;
+import co.alphavantage.output.exchange.data.CurrencyExchangeData;
+
+public class CurrencyExchangeTest {
+	private ForeignExchange foreignExchange;
+	
+	@Test
+	public void conversionFromEURToUSD(){
+		String json = "{\r\n  \"Realtime Currency Exchange Rate\": {\r\n    \"1. From_Currency Code\": \"EUR\",\r\n    \"2. From_Currency Name\": \"Euro\",\r\n    \"3. To_Currency Code\": \"USD\",\r\n    \"4. To_Currency Name\": \"United States Dollar\",\r\n    \"5. Exchange Rate\": \"1.18929963\",\r\n    \"6. Last Refreshed\": \"2017-12-01 23:28:54\",\r\n    \"7. Time Zone\": \"UTC\"\r\n  }\r\n}";

```

Comment:
It would be nice to be able to read the json passed in.

Reply:
In fact I would not maintain jsons inside
Strings. I would put em all in test/resources

Reply:
The reason for having them is to not need to switch files when reading the test, but maybe you are right, having a test resources, with full Json would be good. It is a matter of opinion here. But it should be readable.

Reply:
Agreed. And +1 to the json resources as there is nothing more readable than a real json

Reply:
I think for now it is ok with a string, or we need to make this resources thing for all tests. This is something that could be done in a later stage.

Reply:
I think that @cbmarc  is right better to move the json to a file. We can do the refactor for the other files afterwards.